### ISSUE TITLE
Copyright status of equations

### ISSUE THREAD

**Issue by rougier** [ReScience/ReScience#59]
There is a pending question related to the re-use of equations (https://github.com/ReScience/ReScience-submission/pull/33) of the original paper in the replication. If fair use most certainly allows to re-use some equations of the original paper, we don't know if an author can re-use (i.e. copy them) all the equations in the new paper in order to ease the reading.

@iampritishpatil was kind enough to ask the question on [stack exchange](https://academia.stackexchange.com/questions/94667/what-exactly-would-count-as-copyright-infringement-in-a-replication-paper) but no definitive answer yet.

Does anyone have some answer or pointers on that precise matter?

**Comment by khinsen** [comment]
I just discussed this question with a librarian, whose first comment was "I am not a lawyer". A quick summary of what I learned:

 - Copyright law is intentionally imprecise about the limit between fair use and copyright infringement. It's up to the judges to appreciate each individual case.
 - There are no known cases of scientists having been sued for copyright infringement in their papers, not even in cases of clear plagiarism that lead to disciplinary investigations by universities.
 - Scientists are unlikely to be sued by publishers for copyright infringements in articles because the economic interest is too low and the cost of going to court is rather high. Publishers go after high-volume copyright infringement only (SciHub etc.)
- All that means that the question is likely to remain unresolved for quite a while.

**Comment by rougier** [comment]
I finally received an interesting answer from Elsevier:

```
Our Global Rights group has confirmed that these equations are not copyrighted; 
therefore, you can merely acknowledge the 2012 Journal of Theoretical Biology
as the source of the equations but do not require written permission from Elsevier.
```

**Comment by rougier** [comment]
cc @akdiem @benoit-girard @ChristophMetzner @TiinaManninen

I finally received an interesting answer from Elsevier:

```
Our Global Rights group has confirmed that these equations are not copyrighted; 
therefore, you can merely acknowledge the 2012 Journal of Theoretical Biology
as the source of the equations but do not require written permission from Elsevier.
```

**Comment by benoit-girard** [comment]
Should we consider making a 2nd version of the manuscript with all the equations?

**Comment by akdiem** [comment]
It's great that they got back to you and it's certainly good news for the future. I guess the time it took them to answer it shows that it wasn't a straightforward issue.

**Comment by rougier** [comment]
yes, it's good to know for the future. I don't think it's worth making a second version at this point.
@akdiem by the way, did you receive the PeerJ invitation for the paper? Else I can resend it.

**Comment by akdiem** [comment]
@rougier Yes, but is it vital to sign up for a PeerJ account? I try to avoid having accounts on websites where I don't necessarily need them.

**Comment by rougier** [comment]
I don't know actually. Those who have answered are now marked as "accepted" and you're still marked "invited". But we're in review now. Maybe it'll become necessary only if we get accepted.
